tests/test_free_energy.py

```diff
@@ -217,25 +239,30 @@ def binding_model(ff_params):
 
             temperature = 300.0
             pressure = 1.0
+            beta = 1 / (constants.BOLTZ * temperature)
+            endpoint_correct = False
 
             integrator = LangevinIntegrator(temperature, 1.5e-3, 1.0, masses, seed)
 
             barostat = MonteCarloBarostat(x0.shape[0], pressure, temperature, group_idxs, 25, seed)
 
             model = estimator.FreeEnergyModel(
                 unbound_potentials,
+                endpoint_correct,
                 client,
                 host_box,
                 x0,
                 v0,
                 integrator,
+                barostat,
                 lambda_schedule,
                 equil_steps,
                 prod_steps,
-                barostat,
+                beta,
+                "prefix",
```

Comment:
nit: this seems like a lot of positional args to keep track of. Consider using kwargs?